.github/workflows/ci.yml

```diff
@@ -61,6 +61,8 @@ jobs:
           - ember-default-with-jquery
           - embroider-safe
           - embroider-optimized
+          - ember-modifier-2.x
+          - ember-modifier-3.1
```

Comment:
@And012 this should include also scenarios `ember-modifier@3.2` and `ember-modifier@^4.0.0-beta.0` e.g. like its done in https://github.com/adopted-ember-addons/ember-keyboard/blob/master/.github/workflows/ci.yml#L133-L137

that way, we test compatibility with 2.x version, with 3.1 version and with 3.2 (where new API introduced and old API deprecated) as well as compatibility with 4.x where old API is removed

Reply:
@And012 also, pay attention to https://github.com/DockYard/ember-in-viewport/blob/master/config/ember-try.js
Here we need to declare each scenario that is added to `ci.yml`.

as a reference, the following scenarios for `ember-try.js` could look like
```javascript
  {
    name: 'ember-modifier-2.1',
    npm: {
      dependencies: {
        'ember-modifier': '^2.1.2',
      },
    },
  },
  {
    name: 'ember-modifier-3.1',
    npm: {
      dependencies: {
        'ember-modifier': '^3.1.0',
      },
    },
  },
  {
    name: 'ember-modifier-3.2',
    npm: {
      dependencies: {
        'ember-modifier': '^3.2.7',
      },
    },
  },
  {
    name: 'ember-modifier-4.0.0-beta.1',
    npm: {
      dependencies: {
        'ember-modifier': '^4.0.0-beta.1',
      },
    },
  }
```